core/schemas/audit.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from typing import ClassVar, Literal
+
+from core import database_arango
+from core.schemas.model import YetiModel
+from pydantic import computed_field
+
+
+class AuditLog(YetiModel, database_arango.ArangoYetiConnector):
+    _collection_name: ClassVar[str] = "auditlog"
+    _type_filter: ClassVar[str | None] = None
+    _root_type: Literal["auditlog"] = "auditlog"
+
+    @computed_field(return_type=Literal["auditlog"])
+    @property
+    def root_type(self):
+        return self._root_type
+
+
```

Comment:
```suggestion

```

Reply:
Fixed in c370b5def51b8ed547b2cdbd0e2d71ac40228828